Fixes #266: Introduce layout commands for sizing and aligning elements

martin-fleck-at opened the pull request:
Action and command to resize given or selected elements to:
- min/max/avg/first/last width or height

Action and command to align given or selected elements to:
- left/center/right for all/first/last selected element(s)
- top/middle/bottom for all/first/last selected element(s)

Use keybindings within Sprotty for some actions
Register actions in the 'Diagram' menu for Theia

### Align
![align](https://user-images.githubusercontent.com/19170971/58326591-8e726300-7e2d-11e9-8ddb-e5458779bc10.png)

### Resize
![resize](https://user-images.githubusercontent.com/19170971/58326593-9205ea00-7e2d-11e9-9891-1c3588f5b34b.png)

tortmayr commented on client/packages/sprotty-client/src/features/layout/layout-commands.ts:
```diff
@@ -0,0 +1,416 @@
+    }
+}
+
+abstract class LayoutElementsCommand extends Command {
```
According to sprotty guidelines, commands should be annotated with `@injectable`

martin-fleck-at replied:
Done

tortmayr commented on client/packages/sprotty-client/src/features/layout/layout-commands.ts:
```diff
@@ -0,0 +1,416 @@
+}
+
+export class ResizeElementsCommand extends LayoutElementsCommand {
+    static readonly KIND = 'layout:resize';
```
According to sprotty guidelines, commands should be annotated with `@injectable`

martin-fleck-at replied:
Done

tortmayr commented on client/packages/sprotty-client/src/features/layout/layout-commands.ts:
```diff
@@ -0,0 +1,416 @@
+    }
+}
+
+export class AlignElementsCommand extends LayoutElementsCommand {
```
According to sprotty guidelines, commands should be annotated with `@injectable`

martin-fleck-at replied:
Done

martin-fleck-at commented:
Hi Tobi, I addressed your feedback regarding the sprotty guidelines in a follow-up commit and created a new issue for the consideration of the overlap setting: https://github.com/eclipsesource/graphical-lsp/issues/271
